app/controllers/score_controller.py

```diff
@@ -0,0 +1,196 @@
+from app.models.user import User
+from app.models.score import Score
+from app.models.exam import Exam
+from app import response, app, db
+from flask import request, jsonify
+import datetime
+from flask_jwt_extended import *
+
+
+def create_score():
+    try:
+        # id = request.form.get('id')
+        user_id = request.form.get("user_id")
+        exam_id = request.form.get("exam_id")
+        score = request.form.get("score")
+
+        new_score = Score(
+            # id=id,
+            user_id=user_id,
+            exam_id=exam_id,
+            score=score,
+        )
+
+        db.session.add(new_score)
+        db.session.commit()
+
+        return response.success(
+            "",
+            f"Score for user with id {user_id} and exam_id {exam_id} successfully added",
+        )
+    except Exception as e:
+        print(e)
+        return response.internal_server_error()
+
+
+def get_all_scores():
+    try:
+        scores = Score.query.all()
+        return jsonify(
+            {
+                "scores": [
+                    {
+                        "id": score.id,
+                        "user_id": score.user_id,
+                        "exam_id": score.exam_id,
+                        "score": score.score,
+                        "created_at": score.created_at,
+                    }
+                    for score in scores
+                ]
+            }
+        )
+    except Exception as e:
```

Comment:
i guess you can change the except Exception like i did in endpoints-exam branch

Reply:
return jsonify({"error": f"An error occurred: {e}"}), 500

Reply:
Is it normal to give the full error message to the user (or at least in this case the FE service)?